examples/Quasiconvex Optimization by Bisection.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Solve the Quasiconvex problem by Convex feasibility and Bisection Search\n",
```

Comment:
Please capitalize consistently: you don't need to capitalize words (Quasiconvex, Convex, Bisection, Search) unless a) they're proper nouns or b) you're writing in title case (in which case all words except conjunctions, determiners, and prepositions should be capitalized).